subprojects/packagefiles/liburing/meson.build

```diff
@@ -0,0 +1,115 @@
+project('liburing', ['c','cpp'],
+        version: run_command('awk', '/Version:/ { print $2 }', 'liburing.spec').stdout().strip(),
```

Comment:
This is not acceptable, calling Unix tools is prohibited for things in WrapDB as they are not portable. The "proper" solution would be to use a `VERSION` file, for which Meson has builtin support.

Reply:
Even for a convenience library wrapping the Linux exclusive io_uring syscall?
But I can understand the concern and am willing to use a different solution if needed.

Reply:
A very simple solution would be to statically code it, and update it, together with the version number declared in subprojects/liburing.wrap

Reply:
> A very simple solution would be to statically co

This is what I would do. But this means having two points of truth and the call to awk is what the upstream make-based buildsystem does.

Reply:
Well, apparently the redhat spec file says the version is 0.7 and the git tag says the version is 2.0, so color me very confused.

Reply:
I will ask upstream what version should be used where and why ;)

Reply:
https://github.com/axboe/liburing/issues/405

Reply:
Upstream changed their version in the spec file after their liburing-2.0 git tag with [1815337f7dbb](https://github.com/axboe/liburing/commit/1815337f7dbb530ae3e3bc1175d8da214cf76eb5). And since I used the git tag as base for the meson changes we had this version mismatch.